### DISCUSSION TITLE
How can I add a static prefix to a random string?

### DISCUSSION THREAD

**Discussion by thislooksfun** [dubzzz/fast-check#1548] [Q&A] [answered]
I'm trying to generate random hex color strings, so I need something of the form `#hhhhhh`. I have the latter part working via `fc.hexaString({ minLength: 6, maxLength: 6 })` so now I need to prefix it with a `'#'`, but I can't find a way to do so. Am I missing something?

**Comment by dubzzz** [comment, marked as answer]
Something like the snippet below should do what you are looking for:
```js
fc.hexaString({ minLength: 6, maxLength: 6 })
 .map(s => `#${s}`)
```
